pizza_app/img/add-icon.svg

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
@viktor-yakubiv gave me a good advice to use tools like [svgo](https://github.com/svg/svgo) for optimizing SVG vector graphics files. It perfectly works with gulp, for example.

Reply:
thanks 😃, I should definitely try it

Reply:
@iammiro actually, with gulp you can use [imagemin](https://github.com/imagemin/imagemin) that includes _svgo_ but works with other images as well.